### ISSUE TITLE
Error updating from 0.14.3 to 0.14.4

### ISSUE THREAD

**Issue by evanleck** [evanw/esbuild#1852]
I'm hitting an error when updating from 0.14.3 to 0.14.4:

```
Error: Expected "0.14.4" but got "0.14.3"
```

Here's the relevant section from the debug log:

```
1015 verbose stack Error: command failed
1015 verbose stack     at ChildProcess.<anonymous> (/Users/evan/.asdf/installs/nodejs/16.13.1/.npm/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:64:27)
1015 verbose stack     at ChildProcess.emit (node:events:390:28)
1015 verbose stack     at maybeClose (node:internal/child_process:1064:16)
1015 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
1016 verbose pkgid esbuild@0.14.4
1017 verbose cwd /Users/evan/Code/prepaid/web
1018 verbose Darwin 21.2.0
1019 verbose argv "/Users/evan/.asdf/installs/nodejs/16.13.1/bin/node" "/Users/evan/.asdf/installs/nodejs/16.13.1/.npm/bin/npm" "install"
1020 verbose node v16.13.1
1021 verbose npm  v8.3.0
1022 error code 1
1023 error path /Users/evan/Code/prepaid/web/node_modules/esbuild
1024 error command failed
1025 error command sh -c node install.js
1026 error /Users/evan/Code/prepaid/web/node_modules/esbuild/install.js:96
1026 error     throw new Error(`Expected ${JSON.stringify("0.14.4")} but got ${JSON.stringify(stdout)}`);
1026 error           ^
1026 error
1026 error Error: Expected "0.14.4" but got "0.14.3"
1026 error     at validateBinaryVersion (/Users/evan/Code/prepaid/web/node_modules/esbuild/install.js:96:11)
1026 error     at /Users/evan/Code/prepaid/web/node_modules/esbuild/install.js:242:5
1027 verbose exit 1
1028 timing npm Completed in 4147ms
1029 verbose unfinished npm timer reify 1639504816686
1030 verbose unfinished npm timer reify:build 1639504819785
1031 verbose unfinished npm timer build 1639504819786
1032 verbose unfinished npm timer build:deps 1639504819787
1033 verbose unfinished npm timer build:run:postinstall 1639504819801
1034 verbose unfinished npm timer build:run:postinstall:node_modules/esbuild 1639504819801
1035 verbose code 1
1036 error A complete log of this run can be found in:
1036 error     /Users/evan/.npm/_logs/2021-12-14T18_00_16_435Z-debug-0.log
```

I use [asdf](https://asdf-vm.com) to manage my Node versions but I'm on the latest LTS version and have never seen this particular error before using npm or [pnpm](https://pnpm.io).

**Comment by evanw** [comment]
My publishing script has a bug. I already noticed and I'm working on a fix.

**Comment by evanw** [comment]
This should be fixed in version 0.14.5. Sorry about that.

**Comment by evanleck** [comment]
@evanw thank you!
